README.md

```diff
@@ -8,7 +8,7 @@ The app evaluates the readability of page content by comparing the text color wi
 
 - HTML
 - CSS
-- JAVSCRIPT
+- JAVASCRIPT
```

Comment:
Even though this is spelled correctly now, I don't think it needs to be all caps.
Make sure to use the correct casing here